### DISCUSSION TITLE
Setting other area fills onClick

### DISCUSSION THREAD

**Discussion by kholy87** [img-mapper/img-mapper#13] [Q&A] [answered]
Hi,

I am trying to dynamically set another area (not the area that was clicked) to be filled when specific area is clicked. Since im recreating a physical controller virtually we have indicator lights that I just mapped out as an area so when I click a button I want to be able to specify another area to be filled and highlighted. 

Thanks,

kholy

**Comment by NishargShah** [comment, marked as answer]
You can use `preFillColor` for that

**Reply by kholy87** [reply]
so would I just basically set imagemap.areas[name].preFillColor dynamically?

**Reply by NishargShah** [reply]
yes

**Reply by kholy87** [reply]
Thanks!
